autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.css

```
@@ -1,8 +1,9 @@
 .button-container {
     display: flex;
     flex-direction: column;
-    padding: 8px;
     justify-content: space-between;
+    width: calc(100% - 16px);
```

```suggestion
    width: 100%;
    box-sizing: border-box;
```
`border-box` includes padding and border in the box size calculation